platforms/allwinner-d1/src/lib.rs

```diff
@@ -155,6 +159,43 @@ pub fn kernel_entry(config: mnemos_config::MnemosConfig<PlatformConfig>) -> ! {
     #[cfg(feature = "sharp-display")]
     d1.initialize_sharp_display();
 
+    // #[cfg(feature = "sdcard")]
```

Comment:
should we either remove the feature flag, or put it back?

Reply:
I think adding a `sdcard` feature may come in handy. I still need to check how to do that though.
However, this function right here was just for testing, so I'm going to remove it before finalizing PR.